pkg/kv/db.go

```diff
@@ -679,6 +679,8 @@ func (db *DB) AdminRelocateRange(
 
 // AddSSTable links a file into the Pebble log-structured merge-tree.
 //
+// Requests are sent with the BulkNormalPri admission header priority.
```

Comment:
Consider mentioning that they'll inherit the admission control source from the transaction.

Reply:
which transaction?

Reply:
Ah, misread these as being on `Batch`. Is `Source` set here at all? If not, these won't take effect, since the default source bypasses admission control.

Reply:
I don't think there's any magic on DB so yeah, this is useless. I've pulled it out of this PR and made a separate one to do it in sst_batcher after switching sst_batcher to just make and send its own requests (the plumbing on the kv.DB "helpers" was driving me mad)